### ISSUE TITLE
Segfault in Node.js v14

### ISSUE THREAD

**Issue by bit** [andrewosh/hyperdrive-daemon#47]
```
#
# Fatal error in , line 0
# Check failed: result.second.
#
#
#
#FailureMessage Object: 0x7fff4d88eb80
 1: 0xa95a01  [hyperdrive]
 2: 0x19617d4 V8_Fatal(char const*, ...) [hyperdrive]
 3: 0xe58149 v8::internal::GlobalBackingStoreRegistry::Register(std::shared_ptr<v8::internal::BackingStore>) [hyperdrive]
 4: 0xbc5418 v8::ArrayBuffer::GetBackingStore() [hyperdrive]
 5: 0xa08589 node::Buffer::New(node::Environment*, char*, unsigned long, void (*)(char*, void*), void*) [hyperdrive]
 6: 0xa089f3 node::Buffer::New(v8::Isolate*, char*, unsigned long, void (*)(char*, void*), void*) [hyperdrive]
 7: 0x9fdbdd napi_create_external_buffer [hyperdrive]
 8: 0x7fcb29420934  [/usr/local/lib/node_modules/hyperdrive-daemon/node_modules/fuse-native/prebuilds/linux-x64/node.napi.node]
 9: 0x13203a9  [hyperdrive]
10: 0x1332810  [hyperdrive]
11: 0x1320cb8 uv_run [hyperdrive]
12: 0xa6a334 node::NodeMainInstance::Run() [hyperdrive]
13: 0x9f9651 node::Start(int, char**) [hyperdrive]
14: 0x7fcb2c52e0b3 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
15: 0x992f0c  [hyperdrive]
```

**Comment by mafintosh** [comment]
Thanks. Do you get this by just running the daemon?

**Comment by bit** [comment]
daemon starts up but crashes after some time, more quickly if i to ls ~/Hyperdrive

**Comment by dreampulse** [comment]
Same here. But it works with Node v12

**Comment by mafintosh** [comment]
I can reproduce on Mac, will get a fix out.

**Comment by 1player** [comment]
Same here, though the stack trace looks a little different, don't know if it helps:

```
#
# Fatal error in , line 0
# Check failed: result.second.
#
#
#
#FailureMessage Object: 0x7ffeefbf6450
 1: 0x1000d4cb6 node::NodePlatform::GetStackTracePrinter()::$_3::__invoke() [/usr/local/Cellar/node/14.2.0/bin/node]
 2: 0x100a18f85 V8_Fatal(char const*, ...) [/usr/local/Cellar/node/14.2.0/bin/node]
 3: 0x100378a74 v8::internal::GlobalBackingStoreRegistry::Register(std::__1::shared_ptr<v8::internal::BackingStore>) [/usr/local/Cellar/node/14.2.0/bin/node]
 4: 0x10018db87 v8::ArrayBuffer::GetBackingStore() [/usr/local/Cellar/node/14.2.0/bin/node]
 5: 0x100062fd0 node::Buffer::New(node::Environment*, char*, unsigned long, void (*)(char*, void*), void*) [/usr/local/Cellar/node/14.2.0/bin/node]
 6: 0x100062d67 node::Buffer::New(v8::Isolate*, char*, unsigned long, void (*)(char*, void*), void*) [/usr/local/Cellar/node/14.2.0/bin/node]
 7: 0x10005e89f napi_create_external_buffer [/usr/local/Cellar/node/14.2.0/bin/node]
 8: 0x1042b90f4 napi_macros_init_wrap [/usr/local/lib/node_modules/hyperdrive-daemon/node_modules/fuse-native/prebuilds/darwin-x64/node.napi.node]
 9: 0x1006cf2cc uv__async_io [/usr/local/Cellar/node/14.2.0/bin/node]
10: 0x1006ded52 uv__io_poll [/usr/local/Cellar/node/14.2.0/bin/node]
11: 0x1006cf706 uv_run [/usr/local/Cellar/node/14.2.0/bin/node]
12: 0x1000b5079 node::NodeMainInstance::Run() [/usr/local/Cellar/node/14.2.0/bin/node]
13: 0x10005d02a node::Start(int, char**) [/usr/local/Cellar/node/14.2.0/bin/node]
14: 0x7fff72781cc9 start [/usr/lib/system/libdyld.dylib]
15: 0x9
```

homebrew node v14.2.0

**Comment by mafintosh** [comment]
Should be fixed in the latest version :) (fuse version 2.2.2)
